WaterMinder/Cell/HandbookChallengesViewCell.swift

```diff
@@ -0,0 +1,134 @@
+        
+        contentView.addSubview(rightTextChallenge)
+        rightTextChallenge.translatesAutoresizingMaskIntoConstraints = false
+        
+        contentView.addSubview(rightTextNameChallenge)
+        rightTextNameChallenge.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            leftWaterImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20),
+            leftWaterImageView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 10),
+            leftWaterImageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            leftWaterImageView.widthAnchor.constraint(equalToConstant: 110),
+            
+            rightTextChallenge.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 30),
+            rightTextChallenge.leadingAnchor.constraint(equalTo: leftWaterImageView.trailingAnchor, constant: 30),
+            
+            rightTextNameChallenge.topAnchor.constraint(equalTo: rightTextChallenge.bottomAnchor, constant: 10),
+            rightTextNameChallenge.leadingAnchor.constraint(equalTo: leftWaterImageView.trailingAnchor, constant: 30),
+            rightTextNameChallenge.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10),
+            
+            
+            
+            rightWaterImageView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 20),
+            rightWaterImageView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -10),
+            rightWaterImageView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            rightWaterImageView.widthAnchor.constraint(equalToConstant: 110),
+
+            leftTextChallenge.topAnchor.constraint(equalTo: contentView.topAnchor, constant: 30),
+            leftTextChallenge.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 30),
+            
+            leftTextNameChallenge.topAnchor.constraint(equalTo: leftTextChallenge.bottomAnchor, constant: 10),
+            leftTextNameChallenge.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: 30),
+            leftTextNameChallenge.trailingAnchor.constraint(equalTo: rightWaterImageView.leadingAnchor, constant: -10)
+        ])
+    }
+    
+    func set(_ itemsInWater: ChallengesViewModel) {
+        
+        leftWaterImageView.image = UIImage(named: itemsInWater.leftNamedImageView)
+
+        rightWaterImageView.image = UIImage(named: itemsInWater.rightNamedImageView)
+        
+        contentView.backgroundColor = itemsInWater.backgroundColor
+        
+        leftTextChallenge.text = itemsInWater.leftTextChallenge
+        leftTextChallenge.font = .systemFont(ofSize: 16)
+        leftTextChallenge.textColor = .white
+        
+        rightTextChallenge.text = itemsInWater.rightTextChallenge
+        rightTextChallenge.font = .systemFont(ofSize: 16)
```

Comment:
this has no dependency on viewModel, this set up can be done when u create property